docs/collecting-data/collecting-from-own-applications/scala-tracker/setup-2/index.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
Since we are breaking URLs, I suggest to name this `setup` and not `setup-2`. Maybe for this one we need to add a redirect rule to `redirects.js` just to be on the safe side since it could be a popular link.